Commands/InformationManagement/ResetLabel.cs

```diff
@@ -0,0 +1,51 @@
+﻿#if !ONPREMISES
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using SharePointPnP.PowerShell.CmdletHelpAttributes;
+using SharePointPnP.PowerShell.Commands.Base.PipeBinds;
+
+namespace SharePointPnP.PowerShell.Commands.InformationManagement
+{
+    [Cmdlet(VerbsCommon.Reset, "PnPLabel")]
+    [CmdletHelp("Resets a label/tag on the specified list or library to None", Category = CmdletHelpCategory.InformationManagement, SupportedPlatform = CmdletSupportedPlatform.Online)]
+    [CmdletExample(
+       Code = @"PS:> Reset-PnPLabel  -List ""Demo List""",
+       Remarks = @"This resets an O365 label on the specified list or library to None", SortOrder = 1)]
+    [CmdletExample(
+       Code = @"PS:> Reset-PnPLabel  -List ""Demo List"" -SyncToItems $true",
+       Remarks = @"This resets an O365 label on the specified list or library to None and resets the label on all the items in the list and library except Folders and where the label has been manually or previously automatically assigned", SortOrder = 2)]
+
+    public class ResetListComplianceTag : PnPWebCmdlet
```

Comment:
Class names should match the cmdlet name

Reply:
Good catch. I'l update it.